project.clj

```diff
@@ -18,6 +18,7 @@
   :dependencies [[org.clojure/clojure]
                  [org.clojure/java.jmx]
                  [org.clojure/tools.logging]
+                 [org.clojure/tools.nrepl "0.2.13"]
```

Comment:
What is this needed for now?

Reply:
as I understand it, leiningen's dependencies become intermixed with transitive managed dependencies in the 2.8 series. leiningen require tools.nrepl 0.2.12 and clj-parent brings in 0.2.13. This resolves that conflict to favor the version in clj-parent.